src/utils/defaultLanguageStrings.ts

```diff
@@ -257,7 +257,7 @@ export default {
   PAGE_TITLE_REVIEW: 'Review %1',
   PAGE_TITLE_TEMPLATES: 'Templates',
   PAGE_TITLE_LOOKUP_TABLES: 'Lookup Tables',
-  PAGE_TITLE_OUTCOMES: 'Outcomes',
+  PAGE_TITLE_DATA_DISPLAY: 'Data Display',
```

Comment:
I have done the changes on `core_templates` Pt strings, in case we use that in the future to base another Pt speaking country. 🤭